index.html

```diff
@@ -7526,6 +7526,27 @@ <h5>Protocol and Payload Bindings</h5>
           </p>
         </section>
       </section>
+
+      <section id="profile-relationship">
+        <h4>Relationship to Profiles</h4>
+        <p>
+          The Web of Things also define a profiling mechanism in [[WOT-PROFILES]] that allows the reduction of
+          implementation effort. Profiles are built on top of existing Bindings by constraining the flexibility of TDs.
```

Comment:
I agree this is how Profiles should work in WoT 2.0. However, this arguably isn't true for the set of profiles defined in the Profiles 1.0 specification which you are citing here, which define their own protocol bindings which can be used independent of 1.0 binding templates.

In other words, I think this wording will be fine once there is a WOT-PROFILES-2 specification to cite, with a  set of 2.0 Profiles built on top of 2.0 Bindings, but currently it's not completely true.

Not sure how to fix this in the meantime other than removing the reference.